classes/assign.class.php

```diff
@@ -42,7 +42,12 @@ public function get_onlinetext($itemid) {
 
         $moodletextsubmission = $DB->get_record('assignsubmission_onlinetext',
             array('submission' => $itemid), 'onlinetext');
-        return $moodletextsubmission->onlinetext;
+
+        if (isset($moodletextsubmission->onlinetext)) {
+            return $moodletextsubmission->onlinetext;
+        }
+
+        return false;
```

Comment:
I guess we have a check for boolean where we are calling this function.

Reply:
Was just having this discussion with John on Slack. The check is if !empty. I think returning null would make the most sense from a code perspective so I may change it.